### PR TITLE
[Fixes #196] Replaces use of `Utils.animate/1` with inline `SmartAnimation` calls. 

### LINKED ISSUE #196: Utils.animate/1 never stops running in Livebook
**Describe the issue**
Code cells using `Utils.animate/1` in Livebook never stop computing.

**Reproduction Steps (If Applicable)**
1. Go to 'Arithmetic' section of curriculum in Livebook
1. Scroll down to 'Integers'
1. In the "Your Turn" section, click 'Evaluate' for the code cell (`Utils.animate(:biggest_integer)`
1. Watch as the numbers start increasing and never stop

**Expected behavior**
`Utils.animate/1` can/does stop at some point.

Possible solutions:
- `Utils.stop_animate/1`
- allow toggling animate on/off with `evaluate` and `stop evaluate` buttons?
- ???

**Screenshots**
See it here: https://www.loom.com/share/5e528da9f48a44339d26fc91f34a6c63

Screen Shot 2022-07-29 at 4 26 12 PM

### PR DESCRIPTION
**Pull request by zorn**
**Issue**

Per the discussions of #196, animations in the Arithmetic Livebook would go on forever.

**Fix**

This change introduces inline use of `SmartAnimation` allowing for the animations to be speed up, stopped, or restarted.

**Notes**

This change is also partial work toward a larger goal of eventually removing all calls to `Utils` described in #176 -- allowing the Livebooks to be more portable and less reliant on libraries that live in this repo but might not be shared with an individual `.livemd` file in the future.

### CODE REVIEW

File: reading/arithmetic.livemd [hunk 1]
```diff
@@ -67,7 +67,7 @@ Often you'll use integers for representing ages, days, years, cash values, and m
 
 ### Your Turn
 
-In the Elixir cell below, replace `0` the biggest integer you can think of (until you get bored).
+In the Elixir cell below, replace `1` the biggest integer you can think of (until you get bored).
```
Comment (zorn) [inline reading/arithmetic.livemd:70]: This change is outside the scope of the original issue. Not sure how strict PRs need to be. Happy to extract this if wanted.

Comment (BrooklinJazz) [reply reading/arithmetic.livemd:70]: No problem!
